components/style/properties/shorthand/border.mako.rs

```diff
@@ -246,7 +246,13 @@ pub fn parse_border(context: &ParserContext, input: &mut Parser)
                                 try!(input.expect_delim('/'));
                                 border_image_outset::parse(context, input)
                             }).ok();
-                            Ok((w, o))
+                            
+                            if ${w}.is_none() && ${o}.is_none {
```

Comment:
Looks wrong. Both w and o are not python mako template variables, they're actual Rust variables.